api/client/builder/types_test.go

```diff
@@ -1994,11 +1350,9 @@ func TestEmptyResponseBody(t *testing.T) {
 			epr := &ExecutionPayloadResponse{}
 			require.NoError(t, json.Unmarshal(encoded, epr))
 			pp, err := epr.ParsePayload()
-			require.NoError(t, err)
-			pb, err := pp.PayloadProto()
 			if err == nil {
 				require.NoError(t, err)
-				require.Equal(t, false, pb == nil)
```

Comment:
Can you explain these changes?

Reply:
because payloadProto can't get called anymore if it's already nil in parsepayload, it just errors earlier now because of how it works